gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/MRCompactorJobRunner.java

```diff
@@ -186,7 +186,7 @@
         ImmutableMap.<String, Object>builder()
             .put(RETRY_TIME_OUT_MS, TimeUnit.MINUTES.toMillis(2L))   //Overall retry for 2 minutes
             .put(RETRY_INTERVAL_MS, TimeUnit.SECONDS.toMillis(5L)) //Try to retry 5 seconds
-            .put(RETRY_MULTIPLIER, 2L) // Muliply by 2 every attempt
+            .put(RETRY_MULTIPLIER, TimeUnit.SECONDS.toMillis(2L))
```

Comment:
`RETRY_INTERVAL_MS` appears to be a ceiling on retry pause duration, so a multiplier of 2s, gives the sequence: [2s, 4s, 5s, 5s, 5s, ...] -
from com.github.rholder.retry.WaitStrategies.ExponentialWaitStrategy:
```
        public long computeSleepTime(Attempt failedAttempt) {
            double exp = Math.pow(2, failedAttempt.getAttemptNumber());
            long result = Math.round(multiplier * exp);
            if (result > maximumWait) {
                result = maximumWait;
            }
```
is that what we want or does it merit increase too?

Reply:
god catch. increased maximumWait